addon/models/learner-group.js

```diff
@@ -174,7 +174,7 @@ export default DS.Model.extend({
   }),
   allParentTitles: computed('parent.{title,allParentTitles}', function(){
```

Comment:
Should depend on the `isTopLevelGroup` property.

Reply:
good catch. fixed!